extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/ImporterPlugin.java

```diff
@@ -43,10 +43,6 @@ public void start(BundleContext context) throws Exception {
         ImporterPlugin.instance = this;
         bundleVersion = context.getBundle().getVersion().toString();
         digestStore = new DigestStore(getStateLocation().toFile());
-        Optional<File> bundleFile = FileLocator.getBundleFileLocation(context.getBundle());
-        if (!bundleFile.isPresent()) {
-           throw new IllegalStateException("Failed to get bundle location.");
-        }
```

Comment:
Previously this part is suppose to get `bundleDictionary` [see here](https://github.com/microsoft/vscode-gradle/blob/3.14.0/extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/ImporterPlugin.java#L46-L56). It can be deleted them since we change project architecture